src/Microsoft.ApplicationInsights.AspNetCore/SdkVersionUtils.cs

```diff
@@ -5,20 +5,17 @@
 
     internal class SdkVersionUtils
     {
-        internal const string VersionPrefix = "aspnetCore-";
+        internal const string VersionPrefix = "aspnet5";
```

Comment:
That would be one const with the same name. It would either have one value or the other (without runtime concatination)